tutorials/io/saving_games.rst

```diff
@@ -168,34 +166,31 @@ way to pull the data out of the file as well.
     // dict of relevant variables.
     public void SaveGame()
     {
-        var saveGame = new File();
-        saveGame.Open("user://savegame.save", (int)File.ModeFlags.Write);
+        using var saveGame = FileAccess.Open("user://savegame.save", FileAccess.ModeFlags.Write);
 
         var saveNodes = GetTree().GetNodesInGroup("Persist");
         foreach (Node saveNode in saveNodes)
         {
             // Check the node is an instanced scene so it can be instanced again during load.
-            if (saveNode.Filename.Empty())
+            if (string.IsNullOrEmpty(saveNode.SceneFilePath))
             {
-                GD.Print(String.Format("persistent node '{0}' is not an instanced scene, skipped", saveNode.Name));
+                GD.Print($"persistent node '{saveNode.Name}' is not an instanced scene, skipped");
                 continue;
             }
 
             // Check the node has a save function.
             if (!saveNode.HasMethod("Save"))
             {
-                GD.Print(String.Format("persistent node '{0}' is missing a Save() function, skipped", saveNode.Name));
+                GD.Print($"persistent node '{saveNode.Name}' is missing a Save() function, skipped");
                 continue;
             }
 
             // Call the node's save function.
             var nodeData = saveNode.Call("Save");
 
             // Store the save dictionary as a new line in the save file.
-            saveGame.StoreLine(JSON.Print(nodeData));
+            saveGame.StoreLine(JSON.Stringify(nodeData));
```

Comment:
In the GDScript example, the `JSON.Stringify` invocation is in a separate line and contains an useful comment about what it does.